examples/nlp/sentence_classification_with_bert.py

```diff
@@ -3,6 +3,8 @@
 
 import numpy as np
 from pytorch_transformers import BertTokenizer
+from torch import nn
```

Comment:
why do we need:
```python
from torch import nn
import torch
```

Reply:
please remove

Reply:
need it for `class_weights = torch.FloatTensor(data_desc.class_weights).cuda()`

Reply:
1. we try not to expose pytorch code on the application script.
2. we shouldn't force anything onto cuda() because it'll break on cpu. use .to(self._device) in nemo.backends.pytorch.common.CrossEntropyLoss

Reply:
Made suggested changes.

Reply:
thx